share/wake/lib/system/remote_cache_api.wake

```diff
@@ -969,3 +1083,114 @@ def mkCacheSearchResponse (json: JValue): Result CacheSearchResponse Error =
     obytes
     | Match
     | Pass
+
+## --- Test functions ---
+export def testDisablingCacheWithEndpoints (api: RemoteCacheApi): Result Unit Error =
```

Comment:
I don't know what the standard pattern for test functions is, but unless you're using package-private functions here, it might be better to pull this into a separate file with a separate `*_test` package, to keep the API that little bit cleaner?